### DISCUSSION TITLE
淡出微信 | /cn/2017/05/wechat/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#931] [Comments]
https://yihui.org/cn/2017/05/wechat/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
说实话 QQ 跟微信比起来，我宁愿用 QQ。微信真是把全国人民都给绑架了。

美国不支持微信支付，我说的是在国内的时候，比如网上买东西什么的，扫个码就直接付钱了，不像以前用网银麻烦得要死；打车也是，不用带钱，一切手机搞定。当然比较夸张的是街上烤羊肉串的摊都支持微信支付（还有谣言说现在街头的乞丐都用微信二维码收款了）。相比之下，国内现在在支付方式上甩了美帝好几条街。

> ——原帖发布于 2017-05-27 06:37:23

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我的理解是微信用了 QQ 的巨大群众基础，然后作为第一代精简版的手机聊天软件横空出世，吸引了大家的注意力，然后开始滚雪球，然后把大家都给绑架了。QQ 已经搞得太臃肿了，微信出道时没那么臃肿，用户比较容易上手吧。我觉得微信也就是站在了移动互联网的台风风口被吹起来的猪。

> ——原帖发布于 2017-05-28 06:29:48

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
没错，邮件确实也很分心。我真正想集中精力做事的时候，必须把邮件也关掉。

> ——原帖发布于 2017-05-28 06:25:28

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
有啊。当然，现在是已经沦为文盲了，提笔忘字。更大的问题是习惯了电脑上打字，随时可以增删改，而纸笔写字的方式一般不允许太多的修改，现在再倒回去用纸笔写文字会比较痛苦。

> ——原帖发布于 2017-06-13 02:23:17

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
谢谢。客从何处来？

> ——原帖发布于 2017-07-21 17:51:47

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我对这个不熟，无法评论。

> ——原帖发布于 2017-09-14 14:25:25

[a comment by an automated account is not shown]

**Reply by baihuiqian** [reply]
update: 被我写邮件沟通的男友已经变成前任。

[a comment by an automated account is not shown]

**Comment by yihui** [comment]
谢谢！已经修正。

> ——原帖发布于 2021-02-27 19:30:25
